db/schema.rb

```diff
@@ -24,7 +24,7 @@
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
     t.string   "slug"
-    t.datetime "closes_at"
+    t.datetime "closes_at",      null: false
```

Comment:
This seems like an unrelated and potentially problematic change... it could fail if there are event records with no `closes_at` date, right?